comictaggerlib/pagelisteditor.py

```diff
@@ -43,6 +43,39 @@ def eventFilter(self, obj, event):
                     # print("ChildAdded")
                     self.mysignal.emit("start")
                     return True
+                if event.type() == QtCore.QEvent.KeyPress:
+                    ret = True
+                    if event.key() == QtCore.Qt.Key_A:
+                        self.mysignal.emit("A")
+                    elif event.key() == QtCore.Qt.Key_B:
+                        self.mysignal.emit("B")
+                    elif event.key() == QtCore.Qt.Key_D:
+                        self.mysignal.emit("D")
+                    elif event.key() == QtCore.Qt.Key_E:
+                        self.mysignal.emit("E")
+                    elif event.key() == QtCore.Qt.Key_F:
+                        self.mysignal.emit("F")
+                    elif event.key() == QtCore.Qt.Key_I:
+                        self.mysignal.emit("I")
+                    elif event.key() == QtCore.Qt.Key_L:
+                        self.mysignal.emit("L")
+                    elif event.key() == QtCore.Qt.Key_M:
+                        self.mysignal.emit("M")
+                    elif event.key() == QtCore.Qt.Key_O:
+                        self.mysignal.emit("O")
+                    elif event.key() == QtCore.Qt.Key_P:
+                        self.mysignal.emit("P")
+                    elif event.key() == QtCore.Qt.Key_R:
+                        self.mysignal.emit("R")
+                    elif event.key() == QtCore.Qt.Key_S:
+                        self.mysignal.emit("S")
+                    elif event.key() == QtCore.Qt.Key_X:
+                        self.mysignal.emit("X")
+                    elif event.key() == QtCore.Qt.Key_Space:
+                        self.mysignal.emit(" ")
+                    else:
+                        ret = False
+                    return ret
```

Comment:
Using qactions defined in the ui file will be better

Reply:
Using QActions now